wicket-core/src/main/java/org/apache/wicket/Session.java

```diff
@@ -130,6 +132,15 @@
 	/** Name of session attribute under which this session is stored */
 	public static final String SESSION_ATTRIBUTE_NAME = "session";
 
+	/**
+	 * taken from BidiUtils
```

Comment:
Maybe expand with "Apache Commons XYZ BidiUtils" or even a link to the Git repository ?

Reply:
I'd love to, but the sources seems to be unavailable :(

This SO article https://stackoverflow.com/questions/4192382/is-there-any-way-to-detect-an-rtl-language-in-java
has link to googlecode which is dead :(

I'll change the wording to be `Google Closure Templates BidiUtils` :)

Reply:
done